plugin/evm/validators/manager.go

```diff
@@ -35,11 +35,12 @@ type manager struct {
 
 // NewManager returns a new validator manager
 // that manages the validator state and the uptime manager.
+// Manager is not thread safe and should be used with the chain context locked.
```

Comment:
```suggestion
// Manager is not thread safe and should be used with the VM context locked.
```
(only update if the other suggestion about lock name is applied)